### ISSUE TITLE
MAUI appicon and splash not updating on iOS device

### ISSUE THREAD

**Issue by joeljaisi** [dotnet/maui#14288]
### Description

I changed the default appicon and splash images in my MAUI solution. The changes were effected on Android device but on iOS device, the default .NET appicon and splash images show. Even just changing the Color property in the .csproj file has no effect on iOS device. I s using Visual Studio Version 17.5.3

### Steps to Reproduce

1. Create a new MAUI app
2. Change the appicon and/or splash images. (or just change the Color property of the SplashScreen property in .csproj file.
3. Run or deploy the app to Android Device - the appicon and/or splash images are updated as expected
4. Run or deploy the app to iOS device - the changes are not effected. The default .NET appiocn and splash images still shows,
5. Try to change the name of the appicon and splashscreen svg file, delete app on iOS device, reboot device, clean solution and deplot to iOS device - still appicon and splash images not updated.

### Link to public reproduction project repository

none

### Version with bug

7.0 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS

### Affected platform versions

iOS 16.4

### Did you find any workaround?

None has worked.

**Comment by mattleibow** [comment]
Duplicate of #8685

**Comment by mattleibow** [comment]
From @npostma  https://github.com/dotnet/maui/issues/8685#issuecomment-1383128133

```
I've run this also. This worked for me (mentioned by @kajetan-kazimierczak )

- delete app from device/emulator and clear all data
- clean solution
- reboot device/emulator (this step is important)
- build and deploy app

Before i tried to delete bin/obj and clean the solution. This alone was not enough. Rebooting the emulator after de-installation did the trick.
```

Basically, the OS is caching things. I have a fix that seems to only be landing in .NET 8: https://github.com/dotnet/maui/pull/10812

You can make this work in .NET 7 by using the Link attribute in your csproj be unique each time you update the splash.

**Comment by joeljaisi** [comment]
> From @npostma [#8685 (comment)](https://github.com/dotnet/maui/issues/8685#issuecomment-1383128133)
> 
> > I've run this also. This worked for me (mentioned by @kajetan-kazimierczak )
> > 
> > * delete app from device/emulator and clear all data
> > * clean solution
> > * reboot device/emulator (this step is important)
> > * build and deploy app
> > 
> > Before i tried to delete bin/obj and clean the solution. This alone was not enough. Rebooting the emulator after de-installation did the trick.
> 
> Basically, the OS is caching things. I have a fix that seems to only be landing in .NET 8: #10812
> 
> You can make this work in .NET 7 by using the Link attribute in your csproj be unique each time you update the splash.

@mattleibow  Did you do those steps on a physical device or emulator? Because for me, those steps are not solving the issue on the physical iPhone device

**Comment by vassiliospallis** [comment]
Same issue here. I don't see progress with this except that is fixed in 8.0.0-preview. But I am using .NET 7 (stable) and in iOS and Android, splash screen cannot be customized at all, it shows the .NET purple default splash screen.
